packages/xo-web/src/common/selectors.js

```diff
@@ -270,6 +273,13 @@ export const isAdmin = (...args) => {
   return user && user.permission === 'admin'
 }
 
+export const getIsPoolAdmin = state =>
+  create(
+    args => resolveIds(createGetObjectsOfType('pool')(args)),
+    getCheckPermissions,
+    (poolsIds, check) => some(poolsIds, poolId => check(poolId, 'administrate'))
+  )
```

Comment:
Selectors are meant to be created only once. Here, with `createGetObjectsOfType('pool')`, you're creating a new selector everytime the input selector (`args => resolveIds(createGetObjectsOfType('pool')(args))`) is called.
```js
export const getIsPoolAdmin = create(
  create(createGetObjectsOfType('pool'), _.keys),
  getCheckPermissions,
  (poolIds, check) => some(poolIds, poolId => check(poolId, 'administrate'))
)
```